http_submission/requirements.txt

```diff
@@ -0,0 +1,6 @@
+cython
```

Comment:
Is cython really required? I thought I sucessfully managed to make it an optional dependency, but maybe I missed something...